compute/containers/api-cli/cont-uploading-with-serverless-framework.mdx

```diff
@@ -7,7 +7,7 @@ content:
   paragraph: This page explains how to upload your container using the Serverless.com framework.
 tags: advanced settings
 categories:
-  - compute
+  - serveless
```

Comment:
```suggestion
  - serverless
```

Reply:
do we also need to add serverless to menu-->filters.json?